itou/www/dashboard/tests.py

```diff
@@ -819,8 +819,8 @@ def test_edit_by_siae_with_nir(self):
         self.client.force_login(user)
 
         back_url = reverse("apply:details_for_siae", kwargs={"job_application_id": job_application.id})
-        url = reverse("dashboard:edit_job_seeker_info", kwargs={"job_application_id": job_application.pk})
-        url = f"{url}?back_url={back_url}"
+        url = reverse("dashboard:edit_job_seeker_info", kwargs={"job_seeker_pk": job_application.job_seeker.pk})
```

Comment:
(jusqu’à la fin de la PR)
```suggestion
        url = reverse("dashboard:edit_job_seeker_info", kwargs={"job_seeker_pk": job_application.job_seeker_id})
```